### DISCUSSION TITLE
Export Excel

### DISCUSSION THREAD

**Discussion by WGALVAO2020** [dbeaver/dbeaver#11851] [Q&A] [answered]
Boa tarde,
          Gostaria de saber se existe alguma opção export no formato de arquivo do Excel *.xls ou *.xlsx?

#### System information: 
- Operating system (distribution) and version
- DBeaver version
- Additional extensions

#### Connection specification:
- Database name and version
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)?

**Comment by uslss** [comment, marked as answer]
Hi @WGALVAO2020 ,
with CE version you can use [Office extension](https://github.com/dbeaver/dbeaver/wiki/Optional-extensions#office-integration).
Also, export to excel is available in [Enterprise edition](https://dbeaver.com/) by default

**Reply by RobiABC** [reply]
Hi @uslss,
I'd like to add at my CE version the Office extension... but it's not clear for me how to add it.
Thanks for your help.

**Reply by RobiABC** [reply]
I've found how to do. :-)
